tests/serializer_test.go

```diff
@@ -22,12 +22,36 @@ type SerializerStruct struct {
 	Roles3                 *Roles                 `gorm:"serializer:json;not null"`
 	Contracts              map[string]interface{} `gorm:"serializer:json"`
 	JobInfo                Job                    `gorm:"type:bytes;serializer:gob"`
-	CreatedTime            int64                  `gorm:"serializer:unixtime;type:time"` // store time in db, use int as field type
-	UpdatedTime            *int64                 `gorm:"serializer:unixtime;type:time"` // store time in db, use int as field type
+	CreatedTime            int64                  `gorm:"serializer:unixtime;type:datetime"` // store time in db, use int as field type
+	UpdatedTime            *int64                 `gorm:"serializer:unixtime;type:datetime"` // store time in db, use int as field type
 	CustomSerializerString string                 `gorm:"serializer:custom"`
 	EncryptedString        EncryptedString
 }
 
+type SerializerPostgresStruct struct {
+	gorm.Model
+	Name                   []byte                 `gorm:"json"`
+	Roles                  Roles                  `gorm:"serializer:json"`
+	Roles2                 *Roles                 `gorm:"serializer:json"`
+	Roles3                 *Roles                 `gorm:"serializer:json;not null"`
+	Contracts              map[string]interface{} `gorm:"serializer:json"`
+	JobInfo                Job                    `gorm:"type:bytes;serializer:gob"`
+	CreatedTime            int64                  `gorm:"serializer:unixtime;type:timestamptz"` // store time in db, use int as field type
+	UpdatedTime            *int64                 `gorm:"serializer:unixtime;type:timestamptz"` // store time in db, use int as field type
+	CustomSerializerString string                 `gorm:"serializer:custom"`
+	EncryptedString        EncryptedString
+}
+
+func (*SerializerPostgresStruct) TableName() string { return "serializer_structs" }
+
+func model(s *SerializerStruct) interface{} {
```

Comment:
Can we make the name of this function more specific? like adaptorSerializerModel or switchSerializerModel? 
I have no other questions.